invenio_records_lom/resources/serializers/schemas.py

```diff
@@ -35,6 +37,28 @@ def get_lang(value):
     return value["langstring"]["lang"]
 
 
+def get_related(obj: dict, relation_kind: str, catalog: str = "repo-pid") -> list:
+    """Get dereferenced records that are `relation_kind`-related to `obj`."""
+    results = []
+    for relation in obj["metadata"].get("relation", []):
+        if get_text(relation["kind"]["value"]) != relation_kind:
+            continue
+
+        for identifier in relation["resource"]["identifier"]:
+            if identifier["catalog"] != catalog:
```

Comment:
minor: normally i would not write catalog if i mean catalog_id. catalog seems more of a object than a string